src/bumerang/request/borrowrequest.py

```diff
@@ -1,19 +1,23 @@
 class BorrowRequest:
 
-    def __init__(self, id, title, description, distance, duration):
+    def __init__(self, id, user_id, title, description, distance, duration, request_type):
```

Comment:
Doesn't need to be this PR, but we'll need the timestamp in here for matchmaking in the future